lib/src/widgets/settings_widgets.dart

```diff
@@ -1341,7 +1341,9 @@ class _DropDownSettingsTileState<T> extends State<DropDownSettingsTile<T>> {
                 onChanged: (newValue) => _handleDropDownChange(newValue, onChanged),
                 enabled: widget.enabled,
                 itemBuilder: (T value) {
-                  return Text(widget.values[value]!);
+                  var val = widget.values[value]!;
```

Comment:
@enchance Instead of managing the widget building like this, in this case, we should take an optional itemBuilder function as property, which will do the same except having to manage the rendering of the widget inside the custom widget we delegate the task to developer.